values.schema.json

```diff
@@ -47,13 +47,37 @@
       }
     },
     "keyValue": {
-      "type": "object", "additionalProperties": false,
```

Comment:
This is needed because environment variables can be defined from configMaps and/or Secrets.

Example:
```
    - name: KAFKA_SSL_TRUSTSTORE_PASSWORD
      valueFrom:
        secretKeyRef:
          name: apicurio
          key: user.password
          optional: false
```